src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -284,13 +289,20 @@ class ArrayField extends React.Component {
                       >
                         <div className="row small-collapse">
                           <div className="small-6 left columns">
-                            <button className="float-left">Update</button>
+                            <button
+                              type="button"
```

Comment:
Adding `type="button"` seems like a good call :+1:

Reply:
Grrr actually, adding `type="button"` to this update button breaks submission since it doesn't have an `onclick` handler. I'm gonna set it to `type="submit"` to be more explicit